.github/workflows/playwright.yml

```diff
@@ -42,29 +42,14 @@ jobs:
           python-version: "${{ env.PYTHON_MAX_VERSION }}"
       - name: Setup virtual env
         uses: ./.github/actions/make_init
-      - name: Run make develop
-        run: make develop
       - name: Install playwright
         run: python -m playwright install --with-deps
-      - name: Run make protobuf
-        run: make protobuf
       - name: Run make frontend-fast
         run: make frontend-fast
       - name: Run make playwright
         run: make playwright
-      - name: Check that all screenshot have been committed
-        run: |
-          set -x;
-
-          UNTRACKED_FILE_COUNT=$(git ls-files --others --exclude-standard | grep snapshots | wc -l | bc -l || echo '0')
-          echo "Untracked files count: ${UNTRACKED_FILE_COUNT}"
-          if [[ "${UNTRACKED_FILE_COUNT}" -gt 0 ]]; then
-            echo "Untracked files:";
-            git ls-files --others --exclude-standard | grep snapshots;
-            exit 1;
-          fi
```

Comment:
Does playwright do this step for us? This seems like it'd be something useful to keep around

Reply:
Our playwright logic stores all missing snapshots in the test-results folder and fails the test. So I believe this part isn't required/used anymore. But I will double check that tomorrow.

Reply:
I did a quick test and verified that it works correctly without this step. So it was redundant in the playwright workflow.